### DISCUSSION TITLE
"No files selected" message when trying to run the async conversion?

### DISCUSSION THREAD

**Discussion by lienpockets** [webdriverio/codemod#105] [General]
Hi, just wondering if I am doing something wrong. I am attempting to use the async converter with the TypeScript parser but every time I try to run a file it says "No files selected, nothing to do." I am using tab to autocomplete so I know my path exists, so I'm not sure what the issue could be.

Example:
`npx jscodeshift -t .\node_modules\@wdio\codemod\async --parser=tsx C:\dev\myproject\e2e\mytestfile.spec.ts`

Full output is always something like this:
```
No files selected, nothing to do. 
All done. 
Results:
0 errors
0 unmodified
0 skipped
0 ok
Time elapsed: 0.003seconds
```

**Comment by christian-bromann** [comment]
Can you try using a relative path?

**Reply by Jahnavi-Tirunagari** [reply]
I'm having the same issues and I am using the relative path

**Reply by Sakthivel185** [reply]
I'm trying to migrate from protractor to webdriver io i am also facing the same issue
